### PR TITLE
Animals table

### PR DESCRIPTION
**Pull request by sireclifford**
- create `animal` table
- use sql to insert data into table
- use sql to query for data

Screenshot 2022-09-13 at 4 37 42 AM
Screenshot 2022-09-13 at 4 37 35 AM
Screenshot 2022-09-13 at 4 37 28 AM
Screenshot 2022-09-13 at 4 37 22 AM
Screenshot 2022-09-13 at 4 37 15 AM
Screenshot 2022-09-13 at 4 37 08 AM
Screenshot 2022-09-13 at 4 37 00 AM
Screenshot 2022-09-13 at 4 36 49 AM

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
-);
+ 	id INT NOT NULL PRIMARY KEY,
+    name VARCHAR(100),
+    date_of_birth DATE,
+    escape_attempts INT,
+    neutered BIT,
```
Comment (HAMMAS-SALEEM) [inline schema.sql]: - [x] neutered should have a boolean datatype BIT datatype stores **1 or 0 or Null**. Kindly change **BIT** to **boolean** :blush:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @sireclifford :wave:

## Highlights :1st_place_medal:

:heavy_check_mark: Following GitFlow
:heavy_check_mark: Descriptive PR Title
:heavy_check_mark: Descriptive PR Summary
:heavy_check_mark: Added Queries

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_See Below Comments_

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (sireclifford) [reply schema.sql]: thank you @HAMMAS-SALEEM. Changes has been made

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @sireclifford, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights:
- All requirements completed
- Professional readme

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**